libpod/image.go

```diff
@@ -183,25 +208,79 @@ func (r *Runtime) PushImage(source string, destination string, options CopyOptio
 
 // TagImage adds a tag to the given image
 func (r *Runtime) TagImage(image *storage.Image, tag string) error {
```

Comment:
Please add locking & a validity check on the runtime

Reply:
should it defer runtime.lock and check if the runtime is valid?

Reply:
Yeah, something like https://github.com/kubernetes-incubator/cri-o/blob/master/libpod/runtime.go#L124-L129

Reply:
fixed